api/room/models.py

```diff
@@ -30,6 +39,11 @@ class Room(Base, Utility):
     events = relationship('Events', cascade="all, delete-orphan")
     response = relationship('Response', cascade="all, delete-orphan")
     devices = relationship('Devices', cascade="all, delete-orphan")
+    room_tags = relationship(
```

Comment:
@Bonifase, I have only updated the relationship table, i have not introduced a new field. These tables already existed i.e the room table and tags table already existed. I have performed deletion and insertion of these because the implementation of the relationship table had not been done as had been discussed. Before one added a room to a tag mutation, with this one can create tags separately ad add a tag to a room mutation.